lib/variable.js

```diff
@@ -79,7 +79,7 @@ module.exports = function environmentVariable (name, value) {
         );
       }
 
-      return Boolean(val);
+      return Boolean('false' === val ? 0 : val);
```

Comment:
Nice find! I think in my head I was under the impression that _Boolean_ would parse "true" and "false". I'm just thinking that maybe we should also check for "0" and treat it as false too?